package.json

```diff
@@ -23,15 +25,31 @@
   "license": "MIT",
   "devDependencies": {
     "browserify": "^11.2.0",
+    "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.10.1",
+    "eslint-plugin-prettier": "^3.1.3",
+    "husky": "^4.2.5",
     "karma": "^0.13.10",
     "karma-browserify": "^4.4.0",
     "karma-chrome-launcher": "^2.2.0",
     "karma-firefox-launcher": "^1.0.1",
     "karma-safari-launcher": "^1.0.0",
     "karma-tap": "^1.0.3",
     "karma-tape-reporter": "^1.0.3",
+    "lint-staged": "^10.1.3",
+    "prettier": "2.0.4",
     "tape": "^4.2.1"
   },
   "dependencies": {},
-  "repository": "https://github.com/chmanie/o9n.git"
+  "repository": "https://github.com/chmanie/o9n.git",
```

Comment:
Would you mind using two spaces for these indents to fit with the rest oft the file? Cheers!